### ISSUE TITLE
Wheel convert produces 'None' as a platform name since 0.31.0

### ISSUE THREAD

**Issue by Starcross** [pypa/wheel#234]
I have been creating wheels as part of my build process from an egg. 

Since version 0.31.0 the wheel produced has a platform name of 'None' instead of 'any'. e.g. <packagename>-<version>-py27-none-None 
This means pip will ignore it.

I tried explicitly adding the platform as 'any' to my setup.py, in case this helped but no difference. I have rolled back to 0.30.0 for now

My target platform is both Linux and Windows, but the wheel command is called on Windows

**Comment by agronholm** [comment]
What is the exact file name of the egg you're converting?

**Comment by agronholm** [comment]
@Starcross would you please answer the question so I can get this out of the way?

**Comment by Starcross** [comment]
Sure, it's informerClient-2.0.4.9-py2.7.egg

**Comment by wiggin15** [comment]
This happens for me too, for package "buildout.wheel", when installing as an egg and then converting the egg to wheel. If it helps, I can only reproduce this on Windows.

System information

```
C:\Users\Administrator>systeminfo | findstr /B /C:"OS Name" /C:"OS Version"
OS Name:                   Microsoft Windows Server 2012 Standard
OS Version:                6.2.9200 N/A Build 9200

C:\Users\Administrator>python --version
Python 2.7.8

C:\Users\Administrator>python -c "import wheel; print wheel.__version__"
0.31.0

C:\Users\Administrator>easy_install --version
setuptools 39.1.0 from c:\cygwin\home\administrator\python\lib\site-packages (Python 2.7)
```

Steps to reproduce

```
C:\Users\Administrator>wget https://files.pythonhosted.org/packages/09/35/795bf537989246237167d3448f5688acb6b6ebc7a20cda3fa3afca0303ad/buildout.wheel-0.2.0.tar.gz#sha256=a09f055b6841a959073c31cff407c26a61ebd6ab924ee79c551d9122cb632521
[...]

C:\Users\Administrator>easy_install buildout.wheel-0.2.0.tar.gz
Processing buildout.wheel-0.2.0.tar.gz
Writing c:\users\admini~1\appdata\local\temp\2\easy_install-skryvw\buildout.wheel-0.2.0\setup.cfg
Running buildout.wheel-0.2.0\setup.py -q bdist_egg --dist-dir c:\users\admini~1\appdata\local\temp\2\easy_install-skryvw\buildout.wheel-0.2.0\egg-dist-tmp-kzd41o
warning: no files found matching 'shootout.cfg'
creating c:\cygwin\home\administrator\python\lib\site-packages\buildout.wheel-0.2.0-py2.7.egg
Extracting buildout.wheel-0.2.0-py2.7.egg to c:\cygwin\home\administrator\python\lib\site-packages
Adding buildout.wheel 0.2.0 to easy-install.pth file

Installed c:\cygwin\home\administrator\python\lib\site-packages\buildout.wheel-0.2.0-py2.7.egg
Processing dependencies for buildout.wheel==0.2.0
Finished processing dependencies for buildout.wheel==0.2.0

C:\Users\Administrator>python -m wheel convert c:\cygwin\home\administrator\python\lib\site-packages\buildout.wheel-0.2.0-py2.7.egg

C:\Users\Administrator>dir
[...]
buildout.wheel-0.2.0-py2.7-none-None.whl
[...]
```

**Comment by agronholm** [comment]
I can reproduce this. It'll be fixed for the next release.

**Comment by agronholm** [comment]
Fix released in v0.31.1.
